makefiles/tools/riotboot_serial.inc.mk

```diff
@@ -0,0 +1,13 @@
+RIOTBOOT_SERIAL := $(RIOTTOOLS)/riotboot_serial/riotboot_serial
+
+FLASHER   ?= $(RIOTBOOT_SERIAL)
+FLASHFILE ?= $(HEXFILE)
```

Comment:
Why a hex file?

Reply:
Because it contains the target addresses and is easy to parse

Reply:
I think it should check that the firmware to be flashed is a hex then (should still be overridable in case a user specifies a firmware path)